src/Components/test/E2ETest/package.json

```diff
@@ -6,11 +6,11 @@
   "private": true,
   "scripts": {
     "selenium-standalone": "selenium-standalone",
-    "prepare": "selenium-standalone install"
+    "prepare": "selenium-standalone install --config ..\\..\\..\\Shared\\E2ETesting\\selenium-config.json"
```

Comment:
It's unusual to see Windows-style paths in a `package.json` file. Does it work with `/` too? If so that would be preferable as we may want this to run on other OSes too.